frontend/src/components/guiComponents/AppButton.vue

```diff
@@ -93,7 +93,6 @@ export default {
 .BC-Gov-PrimaryButton:hover,
 .BC-Gov-SecondaryButton:hover {
   opacity: 0.8;
-  text-decoration: underline;
```

Comment:
We use the AppButton in a lot of places - are we sure that we want to remove the underline?